app/frontend/websocket/handlers/latest.js

```diff
@@ -44,7 +48,8 @@ const handleLatest = async (logger, ws, _payload) => {
       accountInfo: JSON.parse(cacheTrailingTradeCommon['account-info']),
       exchangeSymbols: JSON.parse(cacheTrailingTradeCommon['exchange-symbols']),
       publicURL: cacheTrailingTradeCommon['local-tunnel-url'],
-      apiInfo: binance.client.getInfo()
+      apiInfo: binance.client.getInfo(),
+      password: passArray
```

Comment:
This is not gonna work as WebSocket value is public.

Reply:
I didn't understand why it won't gonna work.